HOW/social/process/how/how-v1.md

```diff
@@ -0,0 +1,163 @@
+
+### HOW Tree
+
+Rather than using the pattern of storing and identifying standards as a long numerical list (as in RFCs or EIPs), HOWs are identified by a semantic path in a hierarchy.  This allows their relationship and types to be more readily understood.  The HOW tree is stored directly as a file hierarchy in a git repo. The HOW tree encompasses different types of standards where the type is represented as the directory structure of in the tree.  The initial structure is:
+
+```
+HOW/
+├─ info/             // General guidelines or information
+├─ social/
+│  ├─ process/       // Changes to HOW processes or other organizational/ecosystem processes
+│  │  ├─ how/        // This standard
+├─ technical/
+│  ├─ core/          // Changes to core Holochain protocols
+│  ├─ application/   // Ecosystem and Application standards and conventions
+```
+
+All HOW standards MUST live as leaves of the tree.  The only files that MAY live at the branches in the tree are a `README.md` file describing the purpose of that part of the tree, and any other files related to that description, i.e. images, diagrams etc.
+
+### HOW Format
+
+HOW standards live as markdown file named `how.md` at the leaves of this directory structure.  The format of a HOW standard file consists of a markdown header and a markdown body.  The content of the header and body MUST follow a template specified by:
+1.  merging the `_requirements.md` files that live at each level of the tree including and above the leaf where the HOW is being added
+2.  merging in any additional body sections or headers specified by a status change in a process flow template.
+
+The above format specification allows the templates for parts of the HOW tree to be contextually appropriate for that part of the tree.  For example, `technical/application` standards require a "Reference Implementation" section, what would not be appropriate for a `social/process` or `info` standard.
+
+### HOW Workflow
+
+HOWs transition through three stages: Define, Refine, & Align, each of which may have one or more statuses that formally indicate the steps in the workflow of a HOW for that stage. In the future we intend for there to be many flavors of the stages of this workflow, but this default HOW process specifies just one and the steps for each stage are as follows:
+
+#### Define
+This is the ideation stage, when a standard is first thought up and proposed.  The status for this stage is: `Draft`.  The steps in the stage are:
+
+1. Author(s) fork the HOW repository, and create a new HOW at the appropriate place in the HOW tree.  Author(s) MUST follow the template for that spot in the tree, as well as add this process (`/social/process/how/how-v1.md`) as the `Source` to the header of the HOW.
```

Comment:
I think there should be some guidance for the directory name of the proposal (one or two words, dash separated, gets the point across about what you're proposing, etc).

Reply:
updated